packages/arista_ngfw/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@8.7
```

Let's bump this to 8.8.
```suggestion
    reference: git@8.8
```